setup.py

```diff
@@ -49,6 +49,7 @@
         'kazoo >= 2.0.0',
         'marathon >= 0.8.1',
         'mesos.cli == 0.1.5',
+        'mesos.interface == 0.28.0',
```

Comment:
I lied above. I did change one thing. This was already pinned in requirements.txt so I figure it's safe to add here.